package.json

```diff
@@ -27,6 +27,7 @@
   },
   "devDependencies": {
     "@types/jest": "^28.1.4",
+    "@types/node-fetch": "^2.6.2",
```

Comment:
can you remove the types? ultimately we implemented an `mjs` script so we don't need these

Reply:
ah, true, thanks! I will

Reply:
done